greet_test.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from greet import greet
+
+
+def test_greet():
+    assert greet("Vasya") == "Hello, Vasya"
```

Comment:
this is a great case for `pytest` parametrize https://docs.pytest.org/en/stable/parametrize.html#parametrize-basics
I suggest using it.